src/js/components/Select/SelectContainer.js

```diff
@@ -126,6 +131,29 @@ const SelectContainer = forwardRef(
       return undefined;
     }, [keyboardNavigation]);
 
+    // make sure options are visible in scroll
+    useEffect(() => {
```

Comment:
Something doesn't seem quite right. When I get past 30 or so here, it stops scrolling/keeping the activeIndex in view: https://deploy-preview-5230--sad-tereshkova-173d07.netlify.app/?path=/story/input-select-many-options--many-options

Reply:
Don't see that occurring on TextInput with suggestions: https://deploy-preview-5230--sad-tereshkova-173d07.netlify.app/?path=/story/input-textinput-suggestions--suggestions